vignettes/precompile.R

```diff
@@ -0,0 +1,15 @@
+# Vignettes that have long run times
+
+library("knitr")
+knit(
+  "estimate_infections_options.Rmd.orig",
```

Comment:
Are the paths right? This only works for me locally if I use the full directory, i.e., "./vignettes/etc".

Reply:
I think so. The idea is that this is run inside the `vignettes` folder so the `figures/` folder ends up in the right place.